### PR TITLE
fix: update query conditions for logs

### PR DESCRIPTION
**Pull request by Fine0830**
Update query conditions for logs.
Reference https://github.com/apache/skywalking/issues/6276

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

**Comment by kezhenxu94** [comment]
Besides the changes in this PR, I had to patch this in order to make the UI work after https://github.com/apache/skywalking/pull/6275

```diff
diff --git a/src/graph/fragments/log.ts b/src/graph/fragments/log.ts
index 163c708..0b55ef8 100644
--- a/src/graph/fragments/log.ts
+++ b/src/graph/fragments/log.ts
@@ -47,7 +47,6 @@ export const QueryServiceLogs = {
           endpointId
           traceId
           timestamp
-          isError
           statusCode
           contentType
           content
```

File: src/graph/fragments/log.ts [hunk 1]
```diff
@@ -47,7 +47,6 @@ export const QueryServiceLogs = {
           endpointId
           traceId
           timestamp
-          isError
           statusCode
```
Comment (wu-sheng) [inline src/graph/fragments/log.ts:50]: I think this should be removed too based on today's tag system. @arugal @kezhenxu94

**Review by kezhenxu94 — APPROVED** [review]
The removal part LGTM

**Comment by wu-sheng** [comment]
We wait for the  update of new field removal.

**Pull request merged**